docs/advanced_topics/privacy.md

```diff
@@ -91,3 +102,23 @@ class VideoPage(Page):
 
     password_required_template = 'video/password_required.html'
 ```
+
+## Password validation
+
+Users often choose poor passwords. Wagtail leverages [Django's password validators](django.contrib.auth.password_validation) to ensure provided passwords are strong.
+
+To customize the validators used, set `WAGTAIL_AUTH_PASSWORD_VALIDATORS`:
```

Comment:
this may sound as nitpicking, but I can see people getting confused by this setting. `WAGTAIL_PRIVATE_PAGES_PASSWORD_VALIDATORS` (or similar) would be a lot more explicit

Reply:
Looking back at it with fresh eyes, I think you're right. I can't see any case Wagtail would need to validate passwords which aren't related to auth or these password protections - so let's make it a specific setting.